docs/guide/docs/introduction.md

```diff
@@ -3,35 +3,39 @@ id: introduction
 title: Introduction
 ---
 
-Building bots is something new and sounds like a fun challenge for most developers. To be able to create software that converses in a natural way with humans is an exciting new reality. The problem is that you don’t want to spend hundreds of hours learning and putting together all the infrastructure and boilerplate code required to get a basic conversational backend up and running.
+Building chatbots is a new, exciting challenge for most developers. Creating software that holds a conversation and responds naturally to humans is a thrilling possibility, and Botpress is turning this into reality. 
+
+![Simple Pizza Bot](assets/pizzabot.gif)
```

Comment:
I don't think we need a screenshot at this point

Reply:
Agree, remove the screenshot

Reply:
Alright